### ISSUE TITLE
Service wide posts not showing across all service pages except on refresh 

### ISSUE THREAD

**Issue by josielewis-warren** [FutureNorthants/northants-website#60]
**Frontend or Backend**: Backend

**Description**

If a service wide post is added it only adds it to the Service homepage, not all service pages within that service area like it is supposed to. We have found that any service pages that are republished will show the service wide alert, but they should appear without the need to republish.
**Accessibility issue**
The Title of the Alert is an H2 and it is being placed before the H1 heading on the page. Can this be changed so it not a heading as we cannot have two H1’s either.

**How to reproduce**

Add a service wide post
The post shows on the service home page: 
![Capture 1](https://user-images.githubusercontent.com/82222237/124471573-eb72ce00-dd94-11eb-8e89-65691916b4fa.PNG)
However will not show on the other services pages: 
![Capture 2](https://user-images.githubusercontent.com/82222237/124471614-fcbbda80-dd94-11eb-879f-84de34d6a99b.PNG)
Until they have been refreshed - once refreshed the post is then shown: 
![Capture 3](https://user-images.githubusercontent.com/82222237/124471649-09d8c980-dd95-11eb-872f-40c48e9de31f.PNG)

**Additional context**

Originally submitted by Robert Walker 05/07/21

**Comment by josielewis-warren** [comment]
Investigation - 12hrs
Work - 6hrs- 30hrs
Test - 1hr
**19-43hrs**

**Comment by willwise** [comment]
I've had a look at this in dev and the issue doesn't exist there? has this been resolved already? is it working in UAT?

**Comment by josielewis-warren** [comment]
> I've had a look at this in dev and the issue doesn't exist there? has this been resolved already? is it working in UAT?

@WestNorthantsRob can you take a look in staging please and see if this is still an issue, if you find that it is, can you take @willwise through the issue so he can see where the issue is please?

**Comment by WestNorthantsRob** [comment]
@willwise In trying to test this I now have a different problem.

I added an alert to the Adult Social Services page. When I viewed the live page on staging I got mix of results, some are displaying on the sub pages, some are not and some pages have an Error 500 issue!

https://staging.westnorthants.gov.uk/adult-social-services Has Banner
https://staging.westnorthants.gov.uk/adult-social-services/report-concern-about-adult Error 500
https://staging.westnorthants.gov.uk/adult-social-services/assessment-and-eligibility No Banner
https://staging.westnorthants.gov.uk/adult-social-services/housing-and-care-options Has Banner
